sdk/react/panels/CastDevicesScreen.js

```diff
@@ -0,0 +1,144 @@
+'use strict';
+
+import PropTypes from 'prop-types';
+
+import React from 'react';
+import {Animated, ListView, Text, View, TouchableHighlight} from 'react-native';
+
+import {BUTTON_NAMES} from '../constants';
+
+const Utils = require('../utils');
+const styles = Utils.getStyles(require('./style/castScreenStyles.json'));
+
+var ds = new ListView.DataSource({rowHasChanged: (r1, r2) => r1 === r2});
+
+const dismissButtonSize = 20;
+const castButtonSize = 35;
+
+class CastDevicesScreen extends React.Component {
+  static propTypes = {
+    height: PropTypes.number,
+    width: PropTypes.number,
+    onDismiss: PropTypes.func,
+    config: PropTypes.object,
+    data: PropTypes.array,
+    selectedItem: PropTypes.string
+  };
+
+  componentWillMount() {
+    this.state = {
+      translateY: new Animated.Value(this.props.height),
+      opacity: new Animated.Value(0),
+      dataSource: ds.cloneWithRows(this.props.data),
+      selectedID: -1
+    };
+  }
+
+  componentDidMount() {
+    this.state.translateY.setValue(this.props.height);
+    this.state.opacity.setValue(0);
+
+    Animated.parallel([
+      Animated.timing(
+        this.state.translateY,
+        {
+          toValue: 0,
+          duration: 700,
+          delay: 0
+        }),
+      Animated.timing(
+        this.state.opacity,
+        {
+          toValue: 1,
+          duration: 500,
+          delay: 0
+        }),
+    ]).start();
+  }
+
+  onDismissBtnPress = () => {
+    this.props.onDismiss();
+  };
+
+  onDismissPress = () => {
+    Animated.timing(
+      this.state.opacity,
+      {
+        toValue: 0,
+        duration: 500,
+        delay: 0
+      }
+    ).start(this.onDismissBtnPress);
+  };
+
+  render() {
+    const dismissButton = Utils.renderRectButton(BUTTON_NAMES.DISMISS,
+      styles.iconDismiss,
+      this.props.config.icons.dismiss.fontString,
+      this.onDismissPress, dismissButtonSize,
+      this.props.config.moreDetailsScreen.color,
```

Comment:
The same thing: i think it is necessery to use new color for Cast. What do you think?